### PR TITLE
READY: Multi-put for Riak TS data

### PR DESCRIPTION
**Pull request by lukebakken**

### CODE REVIEW

File: riak/transports/tcp/connection.py [hunk 1]
```diff
@@ -202,6 +202,7 @@ def _connect(self):
                                                         self._timeout)
             else:
                 self._socket = socket.create_connection(self._address)
+            # logging.debug('[TcpConnection] new connection created')
```
Comment (javajolt) [inline riak/transports/tcp/connection.py]: Did you mean to leave this comment in?

**Comment by javajolt** [comment]
Tests of KV and TS both look good.  Once you remove the comment, feel free to merge.
:+1:

File: riak/transports/tcp/connection.py (hunk 1 above)
Comment (lukebakken) [inline riak/transports/tcp/connection.py]: Well, it's useful to confirm that multi operations are actually working & creating multiple TCP connections

Comment (javajolt) [inline riak/transports/tcp/connection.py]: OK, just making sure it was not left in on accident.  Merge away.

**Pull request merged**